internal/dag/builder_test.go

```diff
@@ -1517,7 +1517,7 @@ func TestDAGInsert(t *testing.T) {
 				Fqdn: "example.com",
 			},
 			Routes: []projcontour.Route{{
-				Condition: projcontour.Condition{
+				Condition: &projcontour.Condition{
```

Comment:
I think this would work ok if Condition wasn't, uh, conditional. As in every Route should have a condition, given the only condition you can have is Prefix at the moment. 

In the future when we make Condition a set, []Condition, then it will be possible to create a route that has no conditions -- matches everything, and we'll have to figure out if we want to permit that, but even then the type will be
```
type Route struct {
    Conditions []Condition // not []*Condition
}
```

Reply:
Right now I have it such that you don't need to specify a condition. If you're delegated `/blog` for instance, seems redundant to have to specify `/` in the child route.

Reply:
Having multiple conditions on the same route would need some discussion. I'm not sure how you would handle multiple paths in the same route.